### ISSUE TITLE
prefix + y not working (yank command line)

### ISSUE THREAD

**Issue by dragonxlwang** [tmux-plugins/tmux-yank#55]
![untitled](https://cloud.githubusercontent.com/assets/15812510/14071519/327309b8-f47b-11e5-9a85-ff9dd7473508.gif)

As shown above, prefix + y doesn't really work for me.

I have looked into the code and I found [this](https://github.com/tmux-plugins/tmux-yank/blob/master/scripts/copy_line.sh#L38):
```
start_tmux_selection() {
	if [ "$TMUX_COPY_MODE" == "vi" ]; then
		# vi copy mode
		tmux send-key 'Space'
	else
		# emacs copy mode
		tmux send-key 'C-Space'
	fi
}
```

and in my .tmux.conf, I have the following setting to make it more vim-ish:

```
# vi-copy config
unbind-key -t vi-copy 'v'
unbind-key -t vi-copy Space
bind-key -t vi-copy 'v' begin-selection
bind-key -t vi-copy 'y' copy-selection
bind-key -t vi-copy Space clear-selection
bind-key -t vi-copy 'C-v' rectangle-toggle
```

It seems that because of my keybindings, the code doesn't work anymore. Is there anyway that tmux can send command instead of keys which might already been overwritten? Another option is to get the keybindings at-hoc?

**Comment by tino** [comment]
I've got kind of the same problem:
![yank](https://cloud.githubusercontent.com/assets/99524/14807970/76be4c4c-0bd5-11e6-950d-f0e496334feb.gif)

I have vi-mode normally enabled with `setw -g mode-keys vi`, and change my prefix to `C-a`.

**Comment by dragonxlwang** [comment]
i've fixed this issue with patch #56
